RED/ViewModels/Modules/GPSViewModel.cs

```diff
@@ -200,6 +228,9 @@ public void ReceivedRovecommMessageCallback(Packet packet, bool reliable)
                 case "GPSSatellites":
                     NumberOfSatellites = packet.Data[0];
                     break;
+                case "RoverDistanceSession":
+                    RoverDistanceTraveled = RoverDistanceStart + packet.Data[0];
```

Comment:
RoveComm sends and receives intergers and not floats. That means that we would convert our integers to floats (the standard is divide by 1000) on the RoveComm end.